learn/on-premise/upgrade/os-upgrade.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "OS Upgrade"
+id: ""
+sidebar_label: "OS Upgrade"
+---
+---
```

Comment:
refer email for contents

Reply:
OS Upgrade will have Below Sections
- Little bit intro. like Why we need this. like Our WME is installed on Host, the below guide os for upgrade OS   or install patch etc. write it well.
- How Data Stored in WME Setup
- What Data can be Backedup
- How to Move All Data to /wm-data directory of Platform Instance
        -  Passivate all Containers, Stop WME, Stop Docker Service
- Backup Process for Various Infra Provider
          -  AWS, Azure, GCP etc 
-  Upgrade Operating System
- Start WME Setup
- Add Capacity for StudioWorkSpace Instances and App Deployment Instances
Manitain same Font and Size for all main headding. sub heading should start reducing fonts.
When you start writing provider specific instructions, the give some intro with some heading. 
I see smalle i when writing Instance.. use capital I
StudioWorkspace Instance / AppDeployment Instance